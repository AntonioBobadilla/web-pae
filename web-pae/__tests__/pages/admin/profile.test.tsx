import Profile from '@/pages/admin/profile';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import React from 'react';

describe('Admin Profile', () => {
  it('renders admin profile page unchanged', () => {
    const { container } = render(<Profile />);
    expect(container).toMatchSnapshot();
  });
});
