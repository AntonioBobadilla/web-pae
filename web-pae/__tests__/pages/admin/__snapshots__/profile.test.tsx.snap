// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin Profile renders admin profile page unchanged 1`] = `
<div>
  <h1>
    PROFILE
  </h1>
</div>
`;

exports[`Home renders admin profile page unchanged 1`] = `
<div>
  <h1>
    PROFILE
  </h1>
</div>
`;

exports[`Home renders profile page unchanged 1`] = `
<div>
  <h1>
    PROFILE
  </h1>
</div>
`;
